docs/CONTRIBUTING.md

```diff
@@ -188,9 +188,7 @@ We use these to test anything related to our scripts (most of which live under `
 
 ### Integration tests
 
-These are a work in progress. We build code-server and run a script called
-[test-standalone-release.sh](../ci/build/test-standalone-release.sh), which
-ensures that code-server's CLI is working.
+These are a work in progress. We build code-server and run tests with `yarn test:integration`, which ensures that code-server's binary works.
```

Comment:
I am not sure exactly how to word this but I think we should avoid `binary` since code-server is a bunch of loose files rather than a single binary.

Maybe...`ensures that code-server builds work on their respective platforms`?